lib/main_common.pm

```diff
@@ -458,7 +458,9 @@ sub load_extra_tests() {
         # dependency of git test
         loadtest "console/sshd";
         loadtest "console/zypper_ref";
-        loadtest "console/zypper_info";
+        if (sle_version_at_least('12-SP2') || check_env('DISTRI', 'opensuse')) {
```

Comment:
I would not exclude zypper_info as a whole. The test module already tries to handle it with the recent PR by @SergioAtSUSE . if it fails it can be improved but within the module